jupyterhub/values.yaml

```diff
@@ -204,3 +204,6 @@ cull:
   users: false
   timeout: 3600
   every: 600
+
+debug:
+  enabled: false
```

Comment:
Is there a reason for this to be `debug.enabled` instead of just `debug: false`?

Reply:
I copied the style used for [`rbac.enabled`](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/e2fbdc247953d17563fb017e4e3d00e66d2c1d1d/jupyterhub/values.yaml#L49)